fresnel/geometry.py

```diff
@@ -281,8 +270,8 @@ def get_extents(self):
         r = self._geometry.getRadius();
         res2d = numpy.array([numpy.min(pos - r, axis=0),
                            numpy.max(pos + r, axis=0)])
-        res = numpy.array([[res2d[0][0], res2d[0][1], 0],
-                           [res2d[1][0], res2d[1][1], numpy.max(self.height[:])]])
+        res = numpy.array([[res2d[0][0], res2d[0][1], -0.1],
```

Comment:
Internally the code uses `-1e-5` to `1e-5` for its boundaries. Can those values be used here, directly or indirectly?